cob_bms_driver/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_bms_driver)
 
+add_compile_options(-std=c++11)
```

Comment:
Don't do this, it will open Pandora's box.

Reply:
using `boost::lexical_cast` instead now...sufficient for your approval?

Reply:
can you elaborate why this is not the way to specify the usage of c++11(14, 17)..?

Reply:
@ipa-mdl suggests not to use c++11 for kinetic at all - abi-compatibility...